RELEASES.md

```diff
@@ -86,7 +86,16 @@ project layouts.**
 > If you _really_ want a system-wide install, you can pass `--break-system-packages` to `pip` outside a virtual enivornment.
 > In this case, commandline interface shims for executables are installed to `/usr/local/bin`, which normally has precedence over `/usr/bin` and might therefore conflict with a previous installation of ROCm.
 
-<!-- TODO: mapping from product name to gfx family -->
+| **Product Name**                   | **GFX Family** |
+| ---------------------------------- | -------------- |
+| MI300A/MI300X                      | gfx94X-dcgpu   |
+| MI350X/MI355X                      | gfx950-dcgpu   |
```

Comment:
(No changes needed now)

I'm expecting this list to grow another ~15 items as we add releases for the rest of https://github.com/ROCm/TheRock/blob/main/cmake/therock_amdgpu_targets.cmake, in which case we'll want to decide on sorting and grouping. Maybe group by [datacenter, consumer, embedded]? Sort by most recently released first?